docker/docker-compose.test.yml

```diff
@@ -18,6 +18,13 @@ services:
       INFLUXDB_HTTP_LOG_ENABLED: 'true'
       INFLUXDB_CONTINUOUS_QUERIES_LOG_ENABLED: 'false'
 
+  rabbitmq:
+    image: rabbitmq:3.6.9
```

Comment:
We use version `3.6.5` in production. https://github.com/metabrainz/docker-server-configs/blob/master/scripts/services.sh#L481